src/commands/sdk.ts

```diff
@@ -252,28 +301,28 @@ export class SdkCommands {
 		vs.commands.executeCommand("vscode.openFolder", projectFolderUri, openInNewWindow);
 	}
 
-	private async createFlutterScreenshot(): Promise<void> {
-		// get screenshot path
-		let screenshotPath = config.flutterScreenshotPath;
+	// private async createFlutterScreenshot(): Promise<void> {
```

Comment:
If this code isn't used anymore, it should be deleted rather than commented out.